projects/demo/src/modules/components/combo-box/examples/1/index.ts

```diff
@@ -0,0 +1,61 @@
+import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import type {TuiStringMatcher} from '@taiga-ui/cdk';
+import {TuiTextfield} from '@taiga-ui/core';
+import {
+    TuiChevron,
+    TuiComboBox,
+    TuiDataListWrapper,
+    TuiFilterByInputPipe,
+} from '@taiga-ui/kit';
+
+const ROMAN_TO_LATIN: Record<string, string> = {
+    I: '1',
+    II: '2',
+    III: '3',
+    IV: '4',
+    V: '5',
+    VI: '6',
+    VII: '7',
+    VIII: '8',
+};
+
+@Component({
+    standalone: true,
+    imports: [
+        FormsModule,
+        TuiChevron,
+        TuiComboBox,
+        TuiDataListWrapper,
+        TuiFilterByInputPipe,
+        TuiTextfield,
+    ],
+    templateUrl: './index.html',
+    encapsulation,
+    changeDetection,
+})
+export default class Example {
+    protected readonly items = [
+        'Charles III',
+        'Elizabeth II',
+        'George VI',
+        'Edward VIII',
+        'George V',
+        'William IV',
+        'George IV',
+        'George III',
+    ];
+
+    protected value: string | null = null;
+
+    protected readonly matcher: TuiStringMatcher<string> = (item, query) => {
+        const romanNumeral = item.split(' ').at(-1)!;
```

Comment:
Are we allowed to use .at?

Reply:
It is not supported by all browsers, yep

It is used only for example's brevity (documentation app code, not library one)